client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-complete/complete-domains-transferred.tsx

```diff
@@ -0,0 +1,73 @@
+import { localizeUrl } from '@automattic/i18n-utils';
+import { useI18n } from '@wordpress/react-i18n';
+import { ResponseDomain } from 'calypso/lib/domains/types';
+
+type Props = {
+	newlyTransferredDomains: ResponseDomain[];
+};
+
+export const CompleteDomainsTransferred = ( { newlyTransferredDomains }: Props ) => {
+	const { __ } = useI18n();
+
+	const formatDate = ( date: string | null ): string => {
```

Comment:
This function can be outside the component, to avoid recreating it at every render.